js/contextMenus.js

```diff
@@ -53,7 +53,7 @@ const addBookmarkMenuItem = (label, siteDetail, closestDestinationDetail, isPare
       if (isParent) {
         siteDetail = siteDetail.set('parentFolderId', closestDestinationDetail && (closestDestinationDetail.get('folderId') || closestDestinationDetail.get('parentFolderId')))
       }
-      windowActions.setBookmarkDetail(siteDetail, siteDetail, closestDestinationDetail, true)
```

Comment:
Oh really? The current behavior is we don't show the location when you are adding from a page because we know the location. So there are only two states: Adding a bookmark (no location) or editing an existing bookmark (location).

Should we make a third state for Adding a bookmark but also showing location? Should we still hide location when you click the star?